moveit_ros/planning/planning_request_adapter_plugins/src/add_time_optimal_parameterization.cpp

```diff
@@ -79,8 +117,16 @@ class AddTimeOptimalParameterization : public planning_request_adapter::Planning
 
     return result;
   }
+  
+  private:
+    double path_tolerance;
+    double resample_dt;
+    double min_angle_change;
```

Comment:
@henningkayser shouldn't these member variables have "_" at end of name?
```suggestion
    double min_angle_change_;
```

Reply:
Yes, they should. It's even enforced by clang-tidy